examples/let-expression/let_expression.bal

```diff
@@ -0,0 +1,47 @@
+import ballerina/io;
+
+public function main() {
+
+    // Following is a simple let expression which involves only a single
+    // variable declaration.
+    int a = let int b = 1 in b * 2;
+    io:println("a: ", a);
+
+    // Declare multiple variables in a let expression.
+    string greeting = let string hello = "Hello ",
+                      string ballerina = "Ballerina!"
```

Comment:
Shall we indent this line for it to be aligned with the let var declaration in the previous line?

Reply:
Fixed.